Signal/src/ViewControllers/AdvancedSettingsTableViewController.m

```diff
@@ -130,21 +129,9 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
         [DDLog flushLog];
         [Pastelog submitLogs];
     } else if ([tableView cellForRowAtIndexPath:indexPath] == self.registerPushCell) {
-        OWSSyncPushTokensJob *syncJob =
-            [[OWSSyncPushTokensJob alloc] initWithPushManager:[PushManager sharedManager]
-                                               accountManager:[Environment getCurrent].accountManager
-                                                  preferences:[Environment preferences]];
-        syncJob.uploadOnlyIfStale = NO;
-        [syncJob run]
-            .then(^{
-                DDLogDebug(@"%@ Successfully ran syncPushTokensJob.", self.tag);
-                SignalAlertView(NSLocalizedString(@"PUSH_REGISTER_SUCCESS", @"Alert title"), nil);
-            })
-            .catch(^(NSError *error) {
-                DDLogError(@"%@ Failed to run syncPushTokensJob with error: %@", self.tag, error);
```

Comment:
What you have now is consolidating code nicely, but I liked giving the user visual feedback here, while just logging it elsewhere. Reason being that this is in response to an *explicit* action the user takes.

e.g. How does the user know "Did it work?"